Consider the following code:
\inputminted{csharp}{\context/question/ArrayIncrementer.cs}

Follow these steps:
\begin{enumerate}
  \item Compile an execute the above code.
  \item Which exception is thrown?
  \item What causes this exception to be thrown?
  \item Use a \texttt{try-catch} construct to \textsl{skip} the iteration that throw this exception.
  \item Is this the correct solution to the problem, and why?
\end{enumerate}

